### DISCUSSION TITLE
有没有可以doh加密解析的功能

### DISCUSSION THREAD

**Discussion by famuserpro** [nxtrace/NTrace-core#81] [Q&A] [answered]
有时候域名本地dns解析出来不对，需要向cfdns通过doh获取记录，不知道有这个功能吗？

**Comment by tsosunchia** [comment, marked as answer]
1.有此功能，请看 `--help` or README。
2.此类问题不适合发在issue区，请发到discussion区。

**Reply by tsosunchia** [reply]
已迁移

**Reply by tsosunchia** [reply]
<img width="454" alt="image" src="https://user-images.githubusercontent.com/59512455/220887703-dbf59940-f1e8-4b15-a80e-9f974caeada1.png">
